webapp/src/app/chat/messages/messages.component.html

```diff
@@ -0,0 +1,42 @@
+<div #chatMessagesContainer class="messages space-y-4 overflow-y-auto" style="max-height: 100%">
+  <div *ngFor="let message of messageHistory" class="flex w-full" [ngClass]="{ 'justify-end': message.sender === 'USER', 'justify-start': message.sender === 'SYSTEM' }">
+    <div class="flex space-x-2 md:w-3/5" [ngClass]="{ 'flex-row-reverse': message.sender === 'USER' }">
+      <div *ngIf="message.sender === 'USER'" class="ml-2 flex flex-col">
```

Comment:
@FelixTJDietrich should we use @If and @For here?

Reply:
Yes, we do not want to use the legacy Angular way. Please be mindful about this when using ChatGPT